File: module_gan/src/app/java/com/huyingbao/module/gan/GanApplication.java
package com.huyingbao.module.gan;

import com.huyingbao.core.common.CommonApp;

import dagger.android.AndroidInjector;
import dagger.android.support.DaggerApplication;

/**
 * @author liujunfeng
 * @date 2019/1/1
 */
public class GanApplication extends CommonApp {
    @Override
    protected AndroidInjector<? extends DaggerApplication> applicationInjector() {
        return DaggerGanComponent.builder().application(this).build();
    }
}

File: module_gan_kotlin/src/app/java/com/huyingbao/module/gan/GanApplication.kt
package com.huyingbao.module.gan

import com.huyingbao.core.common.CommonApp

import dagger.android.AndroidInjector
import dagger.android.support.DaggerApplication

/**
 * Created by liujunfeng on 2019/1/1.
 */
class GanApplication : CommonApp() {
    override fun applicationInjector(): AndroidInjector<out DaggerApplication> {
        return DaggerGanComponent.builder().application(this).build()
    }
}
